packages/client/src/rtc/publisher.ts

```diff
@@ -244,12 +259,19 @@ export class Publisher {
   private onNegotiationNeeded = async () => {
     console.log('AAA onNegotiationNeeded');
     const offer = await this.publisher.createOffer();
-    if (this.isDtxEnabled && offer.sdp) {
-      offer.sdp = offer.sdp.replace(
-        'useinbandfec=1',
-        'useinbandfec=1;usedtx=1',
-      );
+    let sdp = offer.sdp;
+    if (sdp) {
+      if (this.isDtxEnabled) {
+        sdp = enableDtx(sdp);
+      }
+      if (isReactNative()) {
+        if (this.preferredVideoCodec) {
+          sdp = setPreferredCodec(sdp, 'video', this.preferredVideoCodec);
+        }
+        sdp = setPreferredCodec(sdp, 'audio', this.preferredAudioCodec);
+      }
```

Comment:
@oliverlaz do you want to turn on munging for other browsers besides chrome too?